pkg/controller/migplan/predicate.go

```diff
@@ -85,6 +85,19 @@ func (r PlanPredicate) mapRefs(plan *migapi.MigPlan) {
 			Name:      ref.Name,
 		})
 	}
+
+	// hooks
+	for _, hook := range plan.Spec.Hooks {
```

Comment:
Needs a matching section in the `unmapRefs` function to handle the case where reference to a hook is removed: the MigPlan should no longer reconcile in response to MigHook change.

Reply:
Thank you. I'll fix it.

Reply:
Should be fixed, I believe.